backend/clubs/migrations/0046_auto_20201017_1039.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.2 on 2020-10-17 14:39
```

Comment:
This migration is most likely out of date. Delete this migration file, rebase your changes onto master, run `./manage.py makemigrations`, and commit your new migrations file.

Note that this may corrupt your local development database. Delete it (delete the `db.sqlite3` file) and run `./manage.py migrate` and `./manage.py populate` to create a new one.

Reply:
`flake8`, our Python style checker, is complaining about this file and causing the tests to fail. You can see the details [here](https://app.circleci.com/pipelines/github/pennlabs/penn-clubs/2043/workflows/531d3f1c-045a-4fc2-8979-e7d69b58c4d2/jobs/5283).

To fix this, run `black -l100 .`, `isort`, and `flake8` in the backend folder. This will run Python formatters and style checkers to ensure that the code has no issues.